packages/create-sitecore-jss/src/templates/angular/package.json

```diff
@@ -15,16 +15,12 @@
   },
   "scripts": {
     "ng": "ng",
-    "lint:yml": "ts-node ./scripts/lint-yml.ts",
-    "lint": "ng lint <%- appName %> --fix && npm run lint:yml",
+    "lint": "ng lint <%- appName %> --fix",
     "e2e": "ng e2e",
     "jss": "jss",
-    "start": "cross-env-shell JSS_MODE=disconnected \"npm-run-all --serial bootstrap --parallel start:angular start:proxy start:watch-components\"",
     "start:connected": "npm-run-all --serial bootstrap start:angular start:watch-components",
     "build": "npm-run-all --serial bootstrap --serial build:client build:server",
```

Comment:
yea, that makes sense. Initially it was how you describe it. I changed it because, thinking that base template can't be used on its own, so just keep scaffold in each addon.